edelphi/src/main/resources/fi/metatavu/edelphi/liquibase/query-page-templates-test.sql

```diff
@@ -0,0 +1,74 @@
+INSERT INTO 
+  LocalizedEntry 
+VALUES 
+  (50),
+  (51),
+  (52),
+  (53),
+  (54),
+  (55),
+  (56),
+  (57),
+  (58),
+  (59),
+  (60),
+  (61),
+  (62),
+  (100),
+  (101),
+  (102),
+  (103),
+  (104),
+  (105),
+  (106),
+  (107),
+  (108),
+  (109),
+  (110),
+  (111),
+  (112);
+  
+INSERT INTO 
+  LocalizedValue (id, text, locale, entry_id)
+VALUES 
+  (50, 'Text', 'en', 50),
+  (100, 'Text', 'en', 50),
+  (51, 'Expertise', 'en', 51),
+  (101, 'Expertise', 'en', 51),
+  (52, 'Scale 1d', 'en', 52),
+  (102, 'Scale 1d', 'en', 52),
+  (53, 'Scale 2d', 'en', 53),
+  (103, 'Scale 2d', 'en', 53),
+  (54, 'Time Serie', 'en', 54),
+  (104, 'Time Serie', 'en', 54),
+  (55, 'Grouping', 'en', 55),
+  (105, 'Grouping', 'en', 55),
+  (56, 'Multiselect', 'en', 56),
+  (106, 'Multiselect', 'en', 56),
+  (57, 'Ordering', 'en', 57),
+  (107, 'Ordering', 'en', 57),
+  (58, 'Form', 'en', 58),
+  (108, 'Form', 'en', 58),
+  (60, 'Background Information Form', 'en', 60),
+  (110, 'Background Information Form', 'en', 60),
+  (61, 'Collage 2d', 'en', 61),
+  (111, 'Collage 2d', 'en', 61),
+  (62, 'Multiple 2d scales', 'en', 62),
+  (112, 'Multiple 2d scales', 'en', 62);
+
+INSERT INTO 
+  QueryPageTemplate (id, iconName, archived, creator_id, created, lastModifier_id, lastModified, name_id, pageType, description_id)
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) The line length is greater than 80 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=plsql%3ALineLength)